lib/yao/resources/project.rb

```diff
@@ -12,5 +12,26 @@ def domain?
       @data["is_domain"]
     end
 
+    def servers
+      @servers ||= Yao::Server.list(all_tenants: 1).select{|s| s.tenant_id == id }
```

Comment:
最初からtenan IDで絞った方が高速です！
```suggestion
      @servers ||= Yao::Server.list(project_id: id)
```

Reply:
参考:https://docs.openstack.org/api-ref/compute/?expanded=list-servers-detailed-detail#list-servers-detailed